doc/tools.yml

```diff
@@ -194,3 +194,14 @@ z3:
     - 'formal:min'
     - 'formal'
     - 'formal:all'
+
+#---
+
+apicula:
+  url: 'https://hdl.github.io/awesome/items/apicula'
+  pkg:
+    - 'apicula'
+  use:
+    - 'apicula'
+  in:
+    - 'impl'
```

Comment:
It is neither included in `impl`, since `impl.dockerfile` is not modified in this PR.